dropwizard-cassandra/src/main/java/smartthings/dw/cassandra/CassandraConfiguration.java

```diff
@@ -38,6 +38,7 @@
 	private Boolean autoMigrate = false;
 
 	private QueryOptions queryOptions;
+    private PoolConfiguration pooling;
```

Comment:
This file does not match the .editorconfig by a good amount.  The change I'm adding does.  I tried to reformat to match the spaces set in the config file, but it changes almost the whole file and makes this MR confusing.  So left the reformatting alone and just added what I needed.